BitwardenShared/UI/Tools/Send/Send/SendList/SendListProcessor.swift

```diff
@@ -214,13 +212,11 @@ final class SendListProcessor: StateProcessor<SendListState, SendListAction, Sen
                 type: state.type
             )
             for try await sends in result {
-                return sends
+                state.searchResults = sends
             }
         } catch {
             services.errorReporter.log(error: error)
```

Comment:
🤔 Is it ok that nothing is shown to the UI when an error happens?

Reply:
I'll add an alert.